tests/report/common/tests.cpp

```diff
@@ -682,85 +553,34 @@ void test_remote_report()
  * Post conditions:
  *     1. Report must contain specified report data or zeros as report data.
  */
-
-/*
- * Report data parameters scenarios on enclave side:
- *      a. Report data can be NULL.
- *      b. Report data can be < OE_REPORT_DATA_SIZE
- *      c. Report data can be OE_REPORT_DATA_SIZE
- *      d. Report data cannot exceed OE_REPORT_DATA_SIZE
- *
- * Report data is not a parameter on the host side.
- */
 #ifdef OE_BUILD_ENCLAVE
     {
         oe_result_t expected_result = OE_OK;
-        report_size = sizeof(report_buffer);
-        OE_TEST(
-            GetReport_v1(
-                flags, NULL, 0, NULL, 0, report_buffer, &report_size) == OE_OK);
-        ValidateReport(
-            report_buffer, report_size, true, zeros, OE_REPORT_DATA_SIZE);
-
-        report_size = sizeof(report_buffer);
         report_data_size = 16;
         OE_TEST(
-            GetReport_v1(
+            GetReport_v2(
                 flags,
                 report_data,
                 report_data_size,
                 NULL,
                 0,
-                report_buffer,
-                &report_size) == expected_result);
+                &report_buffer_ptr,
```

Comment:
Where is the corresponding call to 
oe_free_report(report_buffer_ptr);
?  I expected to see it at line 583 before exiting the 'if' block

Reply:
FIxed